must-gather/collection-scripts/gather

```diff
@@ -10,11 +10,11 @@ gather_namespaced_resources ${BASE_COLLECTION_PATH} "${SINCE_TIME}"
 gather_clusterscoped_resources ${BASE_COLLECTION_PATH} "${SINCE_TIME}"
 gather_noobaa_resources ${BASE_COLLECTION_PATH} "${SINCE_TIME}"
 
-
-
 if [ "$(oc get storagecluster -n openshift-storage -o go-template='{{range .items}}{{.spec.externalStorage.enable}}{{"\n"}}{{end}}')" == true ]; then
     echo "Skipping the ceph collection as External Storage is enabled"
-else    
+elif [ -z "$(oc get storagecluster -n openshift-storage -o go-template='{{range .items}}{{.metadata.name}}{{"\n"}}{{end}}')" ]; then
```

Comment:
I would suggest we should check ceph cluster instead of storagecluster. Because it is also possible that we do have storagecluster but ocs-operator is not able to create ceph cluster for any reason. So it won't work in that case I guess.

Reply:
We check for cephcluster here https://github.com/openshift/ocs-operator/blob/bc3d8b519860cbc675b81852a234872f2171240b/must-gather/collection-scripts/gather_ceph_resources#L106.
And this is a bp PR. The PR https://github.com/openshift/ocs-operator/pull/982/commits/671b2a247861bdc3e2805b4dab2bed7f7a4f0e9f got merged in master.

Reply:
ok